### DISCUSSION TITLE
MECSim only works in Linux?

### DISCUSSION THREAD

**Discussion by protonea** [kiranvad/pyMECSim#8] [Q&A]
Hi Kiran,
Just be clarify. the MECSim only works for Linux? I have tried to installed it. First tried the "pip install", which apparently I shouldn't have because of the big upper case letters, and then I manage to do the git method. None seem to work when I try to run it. Now looking into the comments to the issues I see that there is a comment by you that the MECSim only runs on linux. I just want to confirm if that is the case. I use mac.
Best

**Comment by kiranvad** [comment]
Hello @protonea ,
I think that is correct, this package only worked on Linux the last time I checked.
I was meaning to fix this but I have moved on from using CV for my research.
I would have to look into this again when time permits. 

Can you provide an error code with a simple example on the Github issues? I will try to see if there's a quick fix from my side.
